### PR TITLE
If slirp4netns fails, return its stderror

### PR DESCRIPTION
**Pull request by rhatdan**
We currently do not get any information about why slirp4netns fails.
This information could be useful to the user, so we should return it
to the user.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@AkihiroSuda @giuseppe PTAL

File: libpod/networking_linux.go [hunk 1]
```diff
@@ -238,7 +243,8 @@ func (r *Runtime) setupRootlessNetNS(ctr *Container) (err error) {
 					continue
 				}
 				if status.Exited() {
-					return errors.New("slirp4netns failed")
+					slurp, _ := ioutil.ReadAll(stderr)
```
Comment (AkihiroSuda) [inline libpod/networking_linux.go]: No reader during running?
Writer may block.

Comment (mheon) [inline libpod/networking_linux.go]: I don't think this works. The pipe is auto-closed when Go detects the process has exited.

**Comment by AkihiroSuda** [comment]
Probably you can just set bytes.Buffer to exec.Cmd.Stderr

**Comment by rhatdan** [comment]
@AkihiroSuda @giuseppe Could one of you guys take over this PR if you have better ideas?

**Comment by giuseppe** [comment]
alternative version here: https://github.com/containers/libpod/pull/4350

**Pull request closed without merging**